getparty.go

```diff
@@ -252,10 +254,9 @@ func (cmd *Cmd) Run(args []string, version string) (err error) {
 	)
 
 	var eg errgroup.Group
-	transport := cleanhttp.DefaultPooledTransport()
-	transport.TLSHandshakeTimeout = time.Duration(cmd.options.Timeout) * time.Second
-	if cmd.options.InsecureSkipVerify {
-		transport.TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
+	transport, err := cmd.getTransport()
```

Comment:
here we should get `pooled = true` transport, see my comment on `getTransport` method below.